R/aggregate.R

```diff
@@ -68,7 +73,7 @@ agg_climate_data <- function(year, data_raster, climate_var, daily_agg, trans =
   # Get layer names (dates)
```

Comment:
Line 66 -- ERA5 data are available for many other variables. We only downloaded prcp, temp and uv to our server. is there a reason we can't make this flexible enough to work with other variables?

Reply:
No, good catch I'll remove that requirement.